snap/influxdb-grafana/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+grafana:
```

Comment:
Should this docker-compose.yml file be using [Version 2](https://docs.docker.com/compose/compose-file/#/version-2) of the Docker Compose file format?